flight/tasks/imu.py

```diff
@@ -48,7 +48,7 @@ async def main_task(self):
             # SATELLITE.IMU.disable()
 
             log_data = {
-                "time": time.time(),
+                "time": int(time.time()),
```

Comment:
CP's time.time() returns already an int (see docs) and is redundant. Adapting the behavior of time for the emulator should be done externally (mock/etc)